test/Maestro.ScenarioTests/ScenarioTests_Clone.cs

```diff
@@ -18,6 +18,7 @@ namespace Maestro.ScenarioTests;
 internal class ScenarioTests_Clone : MaestroScenarioTestBase
 {
     [Test]
+    [Ignore("We no longer use this functionality")]
```

Comment:
should we just delete this test? It may cause some extra work if we change stuff

Reply:
I'd keep it around. Maybe we will depend on the clone again (maybe we will clone repos like that in the PCS again) or maybe I don't know about some users of the `darc clone` command somewhere and we might eventually break them, so I'd just keep it as ignored